variables.tf

```diff
@@ -126,3 +126,9 @@ variable "tfc_default_project" {
   type        = string
   default     = "Default Project"
 }
+
+variable "key_name_format" {
+  description = "Format string to generate key names. The first parameter is the organization, the second is the project name and third is the workspace name"
+  type        = string
+  default     = "%[1]s-%[2]s-%[3]s"
+}
```

Comment:
i don't think we would need this. If the consumer of this module passes `%[1]s-%[2]s` or an invalid format, the logic for format would fail. Can we remove it ? 

@lawliet89  your thoughts ?